minifi-bootstrap/src/test/resources/Working_with_Logs.yml

```diff
@@ -0,0 +1,253 @@
+  max concurrent tasks: 1
+  scheduling strategy: TIMER_DRIVEN
+  scheduling period: 0 sec
+  penalization period: 30 sec
+  yield period: 1 sec
+  run duration nanos: 0
+  auto-terminated relationships list: []
+  Properties:
+    Delete Attributes Expression:
+    log.type: nifi-user-logs
+- name: UpdateAttribute
+  class: org.apache.nifi.processors.attributes.UpdateAttribute
+  max concurrent tasks: 1
+  scheduling strategy: TIMER_DRIVEN
+  scheduling period: 0 sec
+  penalization period: 30 sec
+  yield period: 1 sec
+  run duration nanos: 0
+  auto-terminated relationships list: []
+  Properties:
+    Delete Attributes Expression:
+    log.type: nifi-app-logs
+Connections:
+- name: ''
+  source name: MergeContent
+  source relationship name: merged
+  destination name: PutHDFS
+  max work queue size: 0
+  max work queue data size: 0 MB
+  flowfile expiration: 0 sec
+  queue prioritizer class:
+- name: ''
+  source name: app logs
+  source relationship name:
+  destination name: UpdateAttribute
+  max work queue size: 0
+  max work queue data size: 0 MB
+  flowfile expiration: 0 sec
+  queue prioritizer class:
+- name: ''
+  source name: Tail User Log
+  source relationship name: success
+  destination name: user logs
+  max work queue size: 0
+  max work queue data size: 0 MB
+  flowfile expiration: 0 sec
+  queue prioritizer class:
+- name: ''
+  source name: UpdateAttribute
+  source relationship name: success
+  destination name: MergeContent
+  max work queue size: 0
+  max work queue data size: 0 MB
+  flowfile expiration: 0 sec
+  queue prioritizer class:
+- name: ''
+  source name: Tail App Log
+  source relationship name: success
+  destination name: app logs
+  max work queue size: 0
+  max work queue data size: 0 MB
+  flowfile expiration: 0 sec
+  queue prioritizer class:
+- name: ''
+  source name: UpdateAttribute
+  source relationship name: success
+  destination name: MergeContent
+  max work queue size: 0
+  max work queue data size: 0 MB
+  flowfile expiration: 0 sec
+  queue prioritizer class:
+- name: ''
+  source name: PutHDFS
+  source relationship name: failure
+  destination name: PutHDFS
+  max work queue size: 0
+  max work queue data size: 0 MB
+  flowfile expiration: 0 sec
+  queue prioritizer class:
+- name: ''
+  source name: user logs
+  source relationship name:
+  destination name: UpdateAttribute
+  max work queue size: 0
+  max work queue data size: 0 MB
+  flowfile expiration: 0 sec
+  queue prioritizer class:
+Remote Processing Groups:
+- name: NiFi Flow
+  url: http://localhost:8080/nifi
+  comment: ''
+  timeout: 30 sec
+  yield period: 10 sec
+  Input Ports:
+  - id: f8c66751-0d14-43ea-ac68-1b5491c8f41b
+    name: user logs
+    comment: ''
+    max concurrent tasks: 1
+    use compression: false
+  - id: 70f8fe07-ecff-4ed4-b050-5c8e4e419eec
+    name: app logs
+    comment: ''
+    max concurrent tasks: 1
+    use compression: false
+Provenance Reporting:
```

Comment:
It wasn't a great example anyways, using the templates I ran through successfully instead.